docs/tutorials/create-an-evm-processing-squid.md

```diff
@@ -139,7 +139,7 @@ It is worth pointing out, at this point, that some important auxiliary code like
 
 In order to adapt the template to the scope of this guide, we need to apply a couple of changes:
 
-1. edit the `CHAIN_NODE` constant to the endpoint URL of Astar network (e.g. `wss://astar.api.onfinality.io/public-ws`)
+1. edit the `CHAIN_NODE` constant to the endpoint URL of Astar network (e.g. `https://astar-mainnet.g.alchemy.com/v2/ALCHEMY_API_KEY`. Create your free [Alchemy](https://docs.alchemy.com/reference/astar-api-quickstart) Astar API key for this)
```

Comment:
We usually provide public endpoints so people don't need to signup for anything to test things out. I have not personally tested if the endpoint is broken, but we can change it to one of the other public ones, and suggest Alchemy as an alternative. Like so:

> (e.g. `wss://astar.public.blastapi.io`,alternatively, to avoid rate limits you could [create an Alchemy account](https://docs.alchemy.com/reference/astar-api-quickstart))